Client/Application/NavigationRouter.swift

```diff
@@ -93,11 +96,11 @@ enum NavigationPath {
             return nil
         }
 
-        if urlString.starts(with: "\(scheme)://deep-link"), let deepURL = components.valueForQuery("url"), let link = DeepLink(urlString: deepURL.lowercased()) {
+        if urlString.starts(with: "ecosia://deep-link/trees"), let link = DeepLink(urlString: url.path) {
```

Comment:
@ecotopian Isn't it redundant to have a deep link named `deep-link` ?
Can it make more sense to have a descriptive name for what action is doing?

Reply:
This builds on top of FFs Deeplink logic and my intention was to stay close to this. We haven't really agreed on an URL from product site yet. So we can still change this.

Reply:
Thanks for clarifying 👍

Reply:
changed to `ecosia://join/code-123`